org.ekstep.reviewercomments-1.0/editor/plugin.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Plugin to get comments from portal
+ * @class reviewerComments
+ * @constructor
+ * @extends org.ekstep.contenteditor.basePlugin
+ * @author Revathi P 
+ * @listens stage:select
+ */
+
+org.ekstep.contenteditor.basePlugin.extend({
+    isApiCalled: false,
+    initialize: function () {
+        ecEditor.addEventListener("stage:select", this.initShowComments, this);
+        this.initShowComments();
+    },
+    initShowComments: function (event) {
+        var instance = this;
+        this.callAPI(function () {
+            setTimeout(function () {
+                instance.initializeComments();
+            }, 1000);
+        });
+    },
+    callAPI: function (callback) {
+        var instance = this;
+        if (!instance.isApiCalled) {
+            this.context = org.ekstep.contenteditor.globalContext;
+            if (!ecEditor._.isUndefined(this.context)) {
+                if (!ecEditor._.isUndefined(this.context.contentId) && this.context.contentId != "") {
+                    var instance = this;
+                    var data = {
+                    };
+                    //Do Api call and get the response  
+                    ecEditor.getService(ServiceConstants.CONTENT_SERVICE).getComments(data, function (error, response) {
+                        if (!error) {
```

Comment:
If there is an error then log Telemetry error event & Show proper error/alert message to the user.